src/GLuaFixer/LintSettings.hs

```diff
@@ -37,6 +37,8 @@ data LintSettings =
     , prettyprint_cStyle             :: !Bool
     , prettyprint_rejectInvalidCode  :: !Bool
     , prettyprint_indentation        :: !String
+
+    , log_format                     :: !String
```

Comment:
Since there are only two possible formats (at this time) I think `String` is not the ideal data type here. For such cases, it's better to create a custom data type:

```haskell
data LogFormat = StandardLogFormat | GithubLogFormat 
```

You'd then need a `FromJSON` instance to parse it from the `LintSettings`. Though I'm thinking the lint settings should perhaps be more for use with glualint in an editor. Wouldn't it be better to make the log format a command line parameter? That way the GitHub action could use that parameter to the the logs in the right format, while the linter in the scripter's favourite editor is unaffected.

Reply:
Yes that was what I initially tried, but then I saw, that glualint usually goes in to different "branches" of code via the parameters and uses most of them more or less in a subcommand manner. That's why I added it to the config options, and also because I did not really know how to refactor the parameter logic to be fancy enough to support such a parameter :sweat_smile: 

I also wanted to add an `auto` mode, which tries to figure out which mode to use based on Environment variables (to automatically recognize running in a GitHub action / workflow), but i did not really get that far with using them in my code, as it always had some issues i do not yet understand, as i literally started with Haskell yesterday. So it would be cool to get some feedback on whether you think an `auto` mode would be useful and maybe some tips on how to check if an ENV var is set (as i do not care about its value) and I did not understand on how to evaluate just that (generally on how to check if it is not `Nothing`).